### ISSUE TITLE
Missing corner in BG from julia

### ISSUE THREAD

**Issue by joa-quim** [GenericMappingTools/gmt#3539]
Oddly this misses the LL corner but the same command works well from command line
```
basemap(region=(0,40,20,60), proj=:merc, figsize=10, frame=(annot=:afg, fill=:lightgreen), savefig="miss_pix.png", show=1, Vd=1)
        psbasemap  -R0/40/20/60 -JM10 -Bpa -B+glightgreen -P -K > C:\TMP\GMTjl_tmp.ps
```
![miss_pix](https://user-images.githubusercontent.com/537321/85643963-8b2f5380-b68d-11ea-8581-17374b83f844.png)

**Comment by PaulWessel** [comment]
The green is painted by a 4-point polygon and then fill it.  This cannot be a GMT problem, somehow a issue in psconvert calling gs?

**Comment by joa-quim** [comment]
No, it's in the PS. Apparently a white square is painted at the corner.
[GMTjl_tmp.ps.zip](https://github.com/GenericMappingTools/gmt/files/4828803/GMTjl_tmp.ps.zip)

**Comment by PaulWessel** [comment]
Run gmt clear sessions and try again. Still there?

**Comment by joa-quim** [comment]
This is from Julia classic mode. No sessions.

**Comment by PaulWessel** [comment]
If you think we are painting a small white square then you will need to step through and see where that is happening, cause I'm not buying it.

**Comment by joa-quim** [comment]
Open the ps file in Illustrator

**Comment by PaulWessel** [comment]
I am buying there is a white square, I am not buying GMT placed it there.

**Comment by PaulWessel** [comment]
You'll have to figure out where it comes from if you want me to fix something.

**Comment by seisman** [comment]
Your PS is generated using the following commands:
```
gmt psbasemap  -R0/40/20/60 -JM10 -Bpa -B+glightgreen -P -K --PS_MEDIA=11900x16840 > map.ps
gmt psxy -T -R0/1/0/1 -JX1 -O >> map.ps
```

Compare the map.ps and your PS file, 
```
4c4
< %%Title: GMT v6.1.0_012bf5c_2020.06.24 [64-bit] Document from psbasemap
---
> %%Title: GMT v6.1.0_229d4bd_2020.06.22 [64-bit] [MP] Document from psbasemap
8c8
< %%CreationDate: Wed Jun 24 22:59:44 2020
---
> %%CreationDate: Thu Jun 25 03:19:15 2020
672c672
< %@GMT: gmt psbasemap -R0/40/20/60 -JM10 -Bpa -B+glightgreen -P -K --PS_MEDIA=11900x16840
---
> %@GMT: gmt psbasemap -R0/40/20/60 -JM10 -Bpa -B+glightgreen -P -K C:\TMP\GMTjl_tmp.ps --PS_MEDIA=11900x16840
766c766
< %@GMT: gmt psxy -T -R0/1/0/1 -JX1 -O
---
> %@GMT: gmt psxy -T -R0/1/0/1 -JX1 -O >C:\TMP\GMTjl_tmp.ps
771a772,773
> {1 A} FS
> -472 0 0 472 472 0 3 0 0 SP
```

**Comment by joa-quim** [comment]
Yes, I do that ``psxy -T`` trick to close the ps files and it forces me to use also **-R -J**. But why is it plotting?

**Comment by joa-quim** [comment]
Closing as nothing that can really be done other than use a very small -JX<size>. Also indirectly fixed by #3672
